myhpom/models/user.py

```diff
@@ -1,8 +1,22 @@
 from django.db import models
 from django.contrib.auth.models import User
 from myhpom import validators
+import base64, hashlib
 
-"""Connect a pre_save receiver/hook to the User model, and use this model in all imports"""
+"""
+* Connect a pre_save receiver/hook to the User model, and use this model in all imports
```

Comment:
gah another e302 - I installed flake8 in a hydroshare instance and ran just to be sure i'm not enforcing some rando rule on my system, and its there too:

```
root@3b657ed24655:/hydroshare# flake8 myhpom/models/user.py
myhpom/models/user.py:4:14: E401 multiple imports on one line
myhpom/models/user.py:13:1: E302 expected 2 blank lines, found 1
myhpom/models/user.py:14:41: W291 trailing whitespace
```

Reply:
well at least my updated flake8 agrees with yours!

Reply:
woot!